2009-flu-uk.qmd

```diff
@@ -1 +1,437 @@
+
+
+## Model design
+
+### Transmission model
+
+Similarly to the SIR model, we saw already, the SEIR model divides the population into four compartments:
+
+- **S**usceptible: Individuals at risk of infection.  
+- **E**xposed: Infected individuals who are not yet infectious.  
+- **I**nfectious: Actively transmitting the disease.  
+- **R**ecovered: Immune individuals.  
+
+Additionally, the model incorporates time-varying transmission rates to account for changes in contact patterns during holidays.
+
+The SEIR model is governed by the following system of ordinary differential equations:
+
+$$
+\left\{
+\begin{aligned}
+    \frac{dS}{dt} &= - \beta(t) \frac{S I}{N} \\
+    \frac{dE}{dt} &= \beta(t) \frac{S I}{N} - \gamma E \\
+    \frac{dI}{dt} &= \gamma - \sigma I \\
+    \frac{dR}{dt} &= \sigma I
+\end{aligned}
+\right.
+$$
+
+where:
+
+- $\beta(t)$ is the time-dependent transmission rate, representing the rate at which susceptible individuals become exposed through contact with infectious individuals.
+- $\gamma$ is the rate at which exposed individuals transition to the infectious stage (the inverse of the incubation period).
+- $\sigma$ is the recovery rate, indicating the rate at which infectious individuals recover or are removed from the infectious population.
+- $N$ is the total population size, assumed to remain constant.
+- $S(t)$, $E(t)$, $I(t)$, and $R(t)$ represent the number of susceptible, exposed, infectious, and recovered individuals, respectively, at time $t$. 
+
+We assume that the contact rate is constant outside of the holiday period, and that it drops by 30% during the summer and 15% during the half-term. We have thus:
+
+$$\beta(t) = R_{0}*\sigma * h(t)$$
+
+with $h(t)$ a piece-wise constant function equal to 0.7 during the summer, 0.85 during the half-term and 1 otherwise.
+
+To seed the model we assume that a small proportion of the population is infected - equally shared between the E and I compartments. The initial conditions are thus:
+$$
+\begin{aligned}
+    S(0) &= (1 - 2 \alpha) N \\
+    E(0) &= \alpha N \\
+    I(0) &= \alpha N \\
+    R(0) &= 0 \\
+\end{aligned}
+$$
+
+### Observation model
+
+The fundamental component of our transmission model is the 'I' compartment tracking continuously the number of infectious people in our population (an infection prevalence). However, what we observe through our surveillance system is a fraction of the cumulative incidence, more precisely an estimate of the number of new cases in a period of 7 days (a week).
+
+In our `odin` model we can track the weekly cumulative incidence $Z(t)$ by:
+
+1. adding a differential equation integrating the instantaneous incidence:
+$$\frac{dZ}{dt} = \gamma E$$
+
+2. reseting $Z(t) = 0$ at the beginning of each week to only account for the new cases from the current week.
+
+Then we get for each $t$ that is a multiple of 7 (i.e. when we change week):
+
+$$Z(t) = \int_{t-7}^{t} \gamma E(s) \ ds$$
+i.e. the cumulative incidence over a week.
```

Comment:
```suggestion

i.e. the cumulative incidence over a week.
```

(spaces around display maths bits so it's more obvious how it will be set)

Reply:
Changed